package-lock.json

```diff
@@ -13792,6 +13822,28 @@
     "@popperjs/core": {
       "version": "2.11.2"
     },
+    "@portabletext/react": {
+      "version": "1.0.6",
+      "resolved": "https://registry.npmjs.org/@portabletext/react/-/react-1.0.6.tgz",
+      "integrity": "sha512-j6BprLiwFz3zr1Lo6BxM2sQ1b3g1JIjGwePeuxqSfbBiEYbGXn2izEckMJ02hSa1f7+RCEUJ+Bojvtzz6BBUaw==",
+      "requires": {
+        "@portabletext/toolkit": "^1.0.5",
+        "@portabletext/types": "^1.0.3"
+      }
+    },
+    "@portabletext/toolkit": {
+      "version": "1.0.5",
+      "resolved": "https://registry.npmjs.org/@portabletext/toolkit/-/toolkit-1.0.5.tgz",
+      "integrity": "sha512-kwVOfVoquufDQNmzzhGxji9A+/T+dhLKKOHEHldM1kMUoNzY6wwrt5H/Plnw4xbdE780kkwqJqHxjxoSYr706A==",
+      "requires": {
+        "@portabletext/types": "^1.0.3"
+      }
+    },
+    "@portabletext/types": {
+      "version": "1.0.3",
+      "resolved": "https://registry.npmjs.org/@portabletext/types/-/types-1.0.3.tgz",
+      "integrity": "sha512-SDDsdury2SaTI2D5Ea6o+Y39SSZMYHRMWJHxkxYl3yzFP0n/0EknOhoXcoaV+bxGr2dTTqZi2TOEj+uWYuavSw=="
+    },
```

Comment:
these changes shouldnt be here plz remove

Reply:
oops, sry removed